src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1509,6 +1509,18 @@ groovyscript.wiki.pyrotech.barrel.description=Over time converts a fluid with fo
 groovyscript.wiki.pyrotech.barrel.duration.value=Sets the time required for the recipe to complete
 groovyscript.wiki.pyrotech.barrel.add=Adds recipes in the format `name`, `input1`, `input2`, `input3`, `input4`, `fluidInput`, `fluidOutput`, `duration`
 
+groovyscript.wiki.pyrotech.brick_kiln.title=Refractory Kiln
+groovyscript.wiki.pyrotech.brick_kiln.description=Converts an item into a new one by burning it. Has a chance to fail
+groovyscript.wiki.pyrotech.brick_kiln.burnTime.value=Sets the time required for the recipe to complete
+groovyscript.wiki.pyrotech.brick_kiln.failureChance.value=Sets the chance to fail the recipe
+groovyscript.wiki.pyrotech.brick_kiln.failureOutput.value=Sets the output when the recipe failed
+groovyscript.wiki.pyrotech.brick_kiln.add=Adds recipes in the format `name`, `input`, `output`, `burnTime`, `failureChance`, `failureOutput`
+
+groovyscript.wiki.pyrotech.brick_oven.title=Refractory Oven
+groovyscript.wiki.pyrotech.brick_oven.description=Can cook food
```

Comment:
could this be elaborated on - perhaps indicating it includes recipes from the Furnace Registry, or my favorite "describe what the process is" line. "Can cook food" just seems a little bare.